### ISSUE TITLE
After downloads complete I'm left wondering what conda is doing

### ISSUE THREAD

**Issue by ijstokes** [conda/conda#5287]
TL;DR: provide some visual cues that conda is doing something after downloads finish, lest the user be wondering "are the downloads stuck?" or "is conda stuck?" or "is this normal?"

Details:

I've just done the following on a Win10x64 system:

```
conda create -n rtest -c jmorrison -c r --file condaliste.txt
```

using this file as input:

[condaliste.txt](https://github.com/conda/conda/files/991212/condaliste.txt)

The downloads have all finished (at least I'm pretty sure they have...) and now I'm just left with a blinking cursor.  Here is the last thing that appeared on the screen:

```
r-modelr-0.1.0 100% |###############################| Time: 0:00:00   2.72 MB/s
rpy2-2.8.5-py3 100% |###############################| Time: 0:00:00   2.11 MB/s
r-caret-6.0_73 100% |###############################| Time: 0:00:01   2.71 MB/s
r-rbokeh-0.5.0 100% |###############################| Time: 0:00:00   3.13 MB/s
r-tidyverse-1. 100% |###############################| Time: 0:00:00 709.42 kB/s
r-essentials-1 100% |###############################| Time: 0:00:00 834.36 kB/s


```

And under the `r-essentials` line just a `_` blinking away.  For several minutes with nothing else.

And now I've just started to get some action, with a Windows dialog box popping up saying:

![screenshot 2017-05-10 17 02 51](https://cloud.githubusercontent.com/assets/293525/25921178/8645aa1e-35a2-11e7-89fa-e6e96882c628.png)

Which I've clicked "yes" to about 5 times (comes up again after 10-20 seconds).

So it looks like conda is chugging along doing something, but that was a pretty long wait (again, several minutes) after the downloads finished and before I got any more hints that it was "alive".

Ah, just finished now as I finished this GH issue, so I can include my `conda info -a`:

```bash
(root) C:\Users\builder>conda info -a
    Current conda install:

                   platform : win-64
              conda version : 4.3.17
           conda is private : False
          conda-env version : 4.3.17
        conda-build version : not installed
             python version : 3.6.0.final.0
           requests version : 2.12.4
           root environment : C:\ProgramData\Anaconda3  (read only)
        default environment : C:\ProgramData\Anaconda3
           envs directories : C:\Users\builder\AppData\Local\conda\conda\envs
                          C:\ProgramData\Anaconda3\envs
                          C:\Users\builder\.conda\envs
              package cache : C:\ProgramData\Anaconda3\pkgs
                          C:\Users\builder\AppData\Local\conda\conda\pkgs
               channel URLs : https://conda.anaconda.org/anaconda-fusion/win-64
                          https://conda.anaconda.org/anaconda-fusion/noarch
                          https://repo.continuum.io/pkgs/free/win-64
                          https://repo.continuum.io/pkgs/free/noarch
                          https://repo.continuum.io/pkgs/r/win-64
                          https://repo.continuum.io/pkgs/r/noarch
                          https://repo.continuum.io/pkgs/pro/win-64
                          https://repo.continuum.io/pkgs/pro/noarch
                          https://repo.continuum.io/pkgs/msys2/win-64
                          https://repo.continuum.io/pkgs/msys2/noarch
                config file : C:\Users\builder\.condarc
               offline mode : False
                 user-agent : conda/4.3.17 requests/2.12.4 CPython/3.6.0 Windows/10 Windows/10.0.15063  

# conda environments:
#
mydev                    C:\Users\builder\AppData\Local\conda\conda\envs\mydev
py27base                 C:\Users\builder\AppData\Local\conda\conda\envs\py27base
py27base_take2           C:\Users\builder\AppData\Local\conda\conda\envs\py27base_take2
py36                     C:\Users\builder\AppData\Local\conda\conda\envs\py36
rtest                    C:\Users\builder\AppData\Local\conda\conda\envs\rtest
root                  *  C:\ProgramData\Anaconda3

sys.version: 3.6.0 |Anaconda custom (64-bit)| (defaul...
sys.prefix: C:\ProgramData\Anaconda3
sys.executable: C:\ProgramData\Anaconda3\python.exe
conda location: C:\ProgramData\Anaconda3\lib\site-packages\conda
conda-build: None
conda-env: C:\ProgramData\Anaconda3\Scripts\conda-env.exe
conda-server: C:\ProgramData\Anaconda3\Scripts\conda-server.exe
user site dirs:

CIO_TEST: <not set>
CONDA_DEFAULT_ENV: root
CONDA_ENVS_PATH: <not set>
PATH: C:\ProgramData\Anaconda3\Library\bin;C:\ProgramData\Anaconda3;C:\ProgramData\Anaconda3\Library\mingw-w64\bin;C:\ProgramData\Anaconda3\Library\usr\bin;C:\ProgramData\Anaconda3\Library\bin;C:\ProgramData\Anaconda3\Scripts;C:\oraclexe\app\oracle\product\11.2.0\server\bin;;C:\Program Files\Microsoft MPI\Bin\;C:\ProgramData\Oracle\Java\javapath;C:\WINDOWS\system32;C:\WINDOWS;C:\WINDOWS\System32\Wbem;C:\WINDOWS\System32\WindowsPowerShell\v1.0\;C:\ProgramData\Anaconda3;C:\ProgramData\Anaconda3\Scripts;C:\ProgramData\Anaconda3\Library\bin;C:\Program Files (x86)\Microsoft SQL Server\130\Tools\Binn\;C:\Program Files\Microsoft SQL Server\130\Tools\Binn\;C:\Program Files (x86)\Microsoft SQL Server\130\DTS\Binn\;C:\Program Files\Microsoft SQL Server\130\DTS\Binn\;C:\Program Files (x86)\Microsoft SQL Server\140\Tools\Binn\;C:\Program Files\Microsoft SQL Server\140\Tools\Binn\;C:\Program Files (x86)\Microsoft SQL Server\140\DTS\Binn\;C:\Program Files\Microsoft SQL Server\140\DTS\Binn\;C:\Program Files\Microsoft SQL Server\Client SDK\ODBC\130\Tools\Binn\;C:\Users\builder\AppData\Local\Microsoft\WindowsApps;
PYTHONHOME: <not set>
PYTHONPATH: <not set>

License directories:
    C:\ProgramData\Anaconda3\licenses
    C:\Users\builder\.continuum
    C:\Users\builder\AppData\Roaming\Continuum
License files (license*.txt):
Package/feature end dates:
```

**Comment by kalefranz** [comment]
I agree.  Need to bring some extra UI back in 4.4.  Got the package download summary back.  Still have a few more things to add though.

**Comment by kalefranz** [comment]
All the UI features dropped in conda 4.3 are back (and sometimes better) in conda 4.4.

I'd like to keep this issue open though, or maybe open a new issue.  On windows, calling `delete_trash()` at the end of CIUR operations can take a significant amount of time., causing a delay of returning control to the command prompt.  During which you're left wondering what conda is doing.  Along with the long path length issue (https://github.com/conda/conda/issues/6153), and rm_rf issue (https://github.com/conda/conda/issues/5154) on Windows, this is another Windows quirk that would be awesome to fix.
